### DISCUSSION TITLE
How to use the updates/mods?

### DISCUSSION THREAD

**Discussion by B1ackydecatty** [MinicraftPlus/minicraft-plus-revived#648] [Q&A]
Its not a bug
BUT i am really new to use those updates/mods
here's me being dumb 
![dumb](https://user-images.githubusercontent.com/45589944/49430599-6db64280-f7e6-11e8-88bd-db4d44dfdb03.PNG)

**Comment by B1ackydecatty** [comment]
literally put it in
was it right?
my screen can't show the hunger bar
so i dunno

**Comment by chrisj42** [comment]
I'm going to assume you are referring to replacing the icons.png, aka a texturepack.

does it work if you put the original icons.png back? If so, then check the "modded" icons.png for the hunger icon and make sure it's there and in the same place as the original icons.png. If it is in the same place, try downloading the "modded" icons.png again, maybe it got messed up or something.

If you overwrote the original icons.png, then just download the whole jar again and run it. If it works, try putting in the "modded" icons.png again. If this makes the hunger bar disappear again, then check the "modded" icons.png as described above.

Let me know how this goes.

**Comment by B1ackydecatty** [comment]
I am working on it
It pretty hard to make customize games

**Comment by B1ackydecatty** [comment]
rip accidentally deleted the whole thing

**Comment by B1ackydecatty** [comment]
I am playing Delux edition can't change it to other version because i ain't smart enough tho

**Comment by chrisj42** [comment]
if you want to play the other version, then download the other version. :P

**Comment by B1ackydecatty** [comment]
then how??? 
i am not a dev or smart nerd

**Comment by chrisj42** [comment]
Lol you don't have to be a dev or a smart nerd, you just have to click the link on the playminicraft website. You clicked one of them easily enough. Just click the other one!

**Comment by chrisj42** [comment]
When you get to the releases page, click on the .jar file. That's the actual game.
